### ISSUE TITLE
[BUG] Start error when starting instance of OpenShift Local.

### ISSUE THREAD

**Issue by rostalan** [crc-org/crc-extension#188]
Reposting here from PD issue: https://github.com/containers/podman-desktop/issues/4631#issue-1976003659

![image](https://github.com/crc-org/crc-extension/assets/75624842/885e5d12-7d17-4644-a03e-bca1756ddfa2)

Had problems reproducing this issue at a later date.

### General information

  * OS: Windows 10
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it: No)
  * Running CRC on: Baremetal-Server

## CRC version
```bash
CRC version: 2.28.0+e1f42d
OpenShift version: 4.13.14
Podman version: 4.4.4
```
  
## CRC status
```bash
DEBU CRC version: 2.28.0+e1f42d
DEBU OpenShift version: 4.13.14
DEBU Podman version: 4.4.4
DEBU Running 'crc status'
CRC VM:          Running
MicroShift:      Running (v4.13.14)
RAM Usage:       1.269GB of 3.828GB
Disk Usage:      4.759GB of 16.1GB (Inside the CRC VM)
Cache Usage:     29.51GB
Cache Directory: C:\Users\rosta\.crc\cache
```

## CRC config
```bash
- consent-telemetry                     : no
- preset                                : microshift
```

## Host Operating System
```bash
Host Name:                 DESKTOP-F03JGE2
OS Name:                   Microsoft Windows 10 Education
OS Version:                10.0.19045 N/A Build 19045
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          rosta.lan@seznam.cz
Registered Organization:
Product ID:                00328-00804-40128-AA569
Original Install Date:     07.07.2022, 17:10:33
System Boot Time:          03.11.2023, 12:01:23
System Manufacturer:       ASUS
System Model:              System Product Name
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 151 Stepping 5 GenuineIntel ~2500 Mhz
BIOS Version:              American Megatrends Inc. 1620, 12.08.2022
Windows Directory:         C:\Windows
System Directory:          C:\Windows\system32
Boot Device:               \Device\HarddiskVolume1
System Locale:             en-us;English (United States)
Input Locale:              en-us;English (United States)
Time Zone:                 (UTC+01:00) Belgrade, Bratislava, Budapest, Ljubljana, Prague
Total Physical Memory:     32 581 MB
Available Physical Memory: 16 487 MB
Virtual Memory: Max Size:  75 141 MB
Virtual Memory: Available: 50 911 MB
Virtual Memory: In Use:    24 230 MB
Page File Location(s):     C:\pagefile.sys
                           E:\pagefile.sys
Domain:                    WORKGROUP
Logon Server:              \\DESKTOP-F03JGE2
Hotfix(s):                 16 Hotfix(s) Installed.
                           [01]: KB5029716
                           [02]: KB5028951
                           [03]: KB5003791
                           [04]: KB5011048
                           [05]: KB5012170
                           [06]: KB5015684
                           [07]: KB5029244
                           [08]: KB5014035
                           [09]: KB5014671
                           [10]: KB5015895
                           [11]: KB5016705
                           [12]: KB5020372
                           [13]: KB5022924
                           [14]: KB5026879
                           [15]: KB5028380
                           [16]: KB5005699
Network Card(s):           5 NIC(s) Installed.
                           [01]: Realtek PCIe 2.5GbE Family Controller
                                 Connection Name: Ethernet
                                 DHCP Enabled:    Yes
                                 DHCP Server:     192.168.18.1
                                 IP address(es)
                                 [01]: 192.168.18.133
                                 [02]: fe80::cc59:2cb6:7619:3bef
                           [02]: Bluetooth Device (Personal Area Network)
                                 Connection Name: Bluetooth Network Connection
                                 Status:          Media disconnected
                           [03]: Intel(R) Wi-Fi 6 AX201 160MHz
                                 Connection Name: Wi-Fi
                                 Status:          Media disconnected
                           [04]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (Default Switch)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 172.23.208.1
                                 [02]: fe80::6ec8:5488:fdcd:2f69
                           [05]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (WSL)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 172.25.0.1
                                 [02]: fe80::2188:f091:c434:d1c0
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.
```

### Steps to reproduce

  1. open PD
  2. create podman machine, start it
  3. create openshift instance with microshift preset
  4. start it
  5. it throws an error

### Expected
instance is started

### Actual
instance i configured, not started

### Logs
VM5:2 main ↪️ [openshift-local] Error: failed to expose port 127.0.0.1:2222 -> 192.168.127.2:22: listen tcp 127.0.0.1:2222: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.

    at DaemonCommander.start (C:\Users\rosta\.local\share\containers\podman-desktop\plugins\quayioredhatdeveloperopenshiftlocalextension\dist\extension.js:9367:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async startCrc (C:\Users\rosta\.local\share\containers\podman-desktop\plugins\quayioredhatdeveloperopenshiftlocalextension\dist\extension.js:25386:24)
    at async createCrcVm (C:\Users\rosta\.local\share\containers\podman-desktop\plugins\quayioredhatdeveloperopenshiftlocalextension\dist\extension.js:26661:24)
    at async Object.create (C:\Users\rosta\.local\share\containers\podman-desktop\plugins\quayioredhatdeveloperopenshiftlocalextension\dist\extension.js:26641:13)
    at async C:\Users\rosta\AppData\Local\Programs\podman-desktop\resources\app.asar\packages\main\dist\index.cjs:384:18579
    at async C:\Users\rosta\AppData\Local\Programs\podman-desktop\resources\app.asar\packages\main\dist\index.cjs:382:16958
    at async WebContents.<anonymous> (node:electron/js2c/browser_init:2:88640)

**Comment by cfergeau** [comment]
This is a duplicate of https://github.com/crc-org/crc/issues/3855
